news/2022-06-30-community-contributors-2021.md

```diff
@@ -0,0 +1,101 @@
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/walavouchey.jpg)](https://osu.ppy.sh/users/5773079)
+
+Ever present in the osu-wiki scene, Walavouchey is a relatively recent face who all but skyrocketed into prestige with his fastidious attention to detail as an editor and writer both. The path of a wikifriend is a long and arduous one, fraught with sprawling pull requests and sometimes unimaginably arduous tasks involving rewriting, reformatting, and restructuring lengthy articles and important explanations, and it is in all of this that Walavouchey has simply excelled without question.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/niva.jpg)](https://osu.ppy.sh/users/197805)
+
+With a storied history of contribution to osu! via the Mappers' Guild as a fourth level (maximum) member and an Mappers' Choice Awards 2020 recipient, it wouldn't be unusual to think that only the mapping scene receives the bulk of Niva's attention between his Global Moderator responsibilities and everything else.
+
+It turns out that contentment with that alone was not on the cards, and so Niva delved deep into the arcane arts of maintaining the osu!wiki, rising today to claim his true place as a recognized wikifriend. His efforts on this front are chiefly centred around - you guessed it - mapping and tournament related articles, making hundreds of changes to promote various other community projects and keep things informative for everyone.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/technocoder.jpg)](https://osu.ppy.sh/users/10338558)
+
+The pillar upon which the members of the osu! community who wish to play the current build of osu! (stable) on their Tim Cook-approved Apple computers, Technocoder has well and truly lived up to his name over the years, creating a litany of Wineskins and other assorted tools designed to help get macOS users situated neatly into the game.
+
+As if this were not enough on its own, he is never far from hand in any macOS-related threads, ever ready and willing to help those who need it. His efforts have easily influenced thousands at this point, and we are enormously grateful for them.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/huu.jpg)](https://osu.ppy.sh/users/6044237)
+
+Entrusted with the wardenship of Project Loved by its previous maintainers, huu was saddled with no small task, and has simply excelled admirably in his time heading the project. From organizing mappers, writers, and captains alike, huu has delved right into the thick of things and helped ensure that the legacy of Project Loved was set to continue in the present day, and it is a sure thing to say that it would not be anywhere near as successful as it presently is without his dedication and attention.
```

Comment:
```suggestion
Entrusted with the wardenship of Project Loved by its previous maintainers, huu was saddled with no small task, and has simply excelled admirably in his time heading the project. From organising mappers, writers, and captains alike, huu has delved right into the thick of things and helped ensure that the legacy of Project Loved was set to continue in the present day, and it is a sure thing to say that it would not be anywhere near as successful as it presently is without his dedication and attention.
```